FWIO/RNTuple/plugins/RNTupleOutputModule.cc

```diff
@@ -198,14 +198,35 @@ namespace edm {
 
   void RNTupleOutputModule::writeRun(RunForOutput const& iRun) { file_->writeRun(iRun); }
 
+  namespace {
+    std::vector<edm::ParameterSet> defaultStreamerOverrides() {
+      std::vector<std::string_view> types = {"LHEEventProduct",
+                                             "recoJetIDedmValueMap",
+                                             "recoPhotons",
+                                             "recoGsfElectrons",
+                                             "recoMuons",
+                                             "recoGsfElectrons",
+                                             "TotemRPLocalTrackedmDetSetVector",
+                                             "edmTriggerResults",
+                                             "GlobalAlgBlkBXVector",
+                                             "SimTrackToTPMap"};
```

Comment:
I think it would be better to mark all these types with `rntupleStreamerMode="true"` in the dictionaries instead (possibly via separate PR).